map.go

```diff
@@ -556,7 +556,12 @@ func handleMapCreateError(attr sys.MapCreateAttr, spec *MapSpec, err error) erro
 	if errors.Is(err, unix.EINVAL) && spec.Type == UnspecifiedMap {
 		return fmt.Errorf("map create: cannot use type %s", UnspecifiedMap)
 	}
-	if errors.Is(err, unix.EINVAL) && spec.Flags&sys.BPF_F_NO_PREALLOC > 0 {
+	// LPM trie key size must be a multiple of 8 and not exceed 2048.
```

Comment:
This seems incorrect according to my experience and the doc `https://docs.ebpf.io/linux/map-type/BPF_MAP_TYPE_LPM_TRIE/`.